samples/Samples.SqlServer/Samples.SqlServer.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net452;net46;net47</TargetFrameworks>
```

Comment:
We should probably add a `netcoreapp2.0` target so we can test this on .NET Core. We'll need to add a conditional package reference to `System.Data.SqlClient` below:
```xml
<ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
  <ReferPackageReferenceence Include="System.Data.SqlClient" Version="4.5.1"/>
</ItemGroup>
```